src/inventory_client/inventory_client.go

```diff
@@ -81,41 +87,27 @@ func (c *inventoryClient) GetCluster() (*models.Cluster, error) {
 	return cluster.Payload, nil
 }
 
-func (c *inventoryClient) GetEnabledHostsNamesIds() (map[string]string, error) {
-	namesIdsMap := make(map[string]string)
+func (c *inventoryClient) GetEnabledHostsNamesHosts(includeIps bool) (map[string]EnabledHostData, error) {
```

Comment:
lets' remove boolean and always return the ips, since we already parsed the json string

Reply:
ok